pyproject.toml

```diff
@@ -70,7 +70,22 @@ exclude = [
 ]
 
 # Black default, although irrelevant with E501 ignored
+[tool.black]
 line-length = 88
+include = '\.pyi?$'
+exclude = '''
+/(
+    \.git
+  | \.hg
+  | \.mypy_cache
+  | \.tox
+  | \.venv
+  | _build
+  | buck-out
+  | build
+  | dist
+)/
+'''
```

Comment:
This should be reverted with the removal of Black. It also (I think accidentally) changed the Ruff configuration - the `line-length = 88` is part of the Ruff configuration and so needs to remain under the `[tool.ruff]` heading.

Reply:
Thanks for the clarification.
I reverted it.